include/sot/core/device.hh

```diff
@@ -22,20 +22,20 @@
 #include <dynamic-graph/all-signals.h>
 #include <dynamic-graph/entity.h>
 
+#include <sot/core/abstract-sot-external-interface.hh>
 #include <sot/core/matrix-geometry.hh>
 
 #include "sot/core/api.hh"
-#include "sot/core/periodic-call.hh"
 
 namespace dynamicgraph {
 namespace sot {
 
 /// Define the type of input expected by the robot
 enum ControlInput {
-  CONTROL_INPUT_NO_INTEGRATION = 0,
-  CONTROL_INPUT_ONE_INTEGRATION = 1,
-  CONTROL_INPUT_TWO_INTEGRATION = 2,
-  CONTROL_INPUT_SIZE = 3
+  POSITION_CONTROL = 0,
+  VELOCITY_CONTROL = 1,
+  TORQUE_CONTROL = 2,
+  CONTROL_SIZE = 3
```

Comment:
This will break https://github.com/stack-of-tasks/sot-torque-control/blob/master/src/device-torque-ctrl.cpp#L83

Reply:
I think this package will also need reworking. @olivier-stasse : what do you think ?

Reply:
I would go for it.